developers.libra.org/scripts/build_docs.sh

```diff
@@ -26,8 +28,19 @@ function install_rustup {
   fi
 }
 
+function install_python {
+  echo "Installing Python 3......"
+  if python3 --version &>/dev/null; then
+    echo "Python 3 is already installed"
+  else
+    echo "Install Python 3 from https://www.python.org/"
+    exit 1
+  fi
+}
+
 BUILD_STATIC=false
 BUILD_RUSTDOCS=false
+BUILD_PYTORCH_SDK_DOCS=false
```

Comment:
pytorch? Do you mean pylibra or something else?

Reply:
LOL - I always do that. Yes, it should be `PYTHON`